File: content/learn/arrays-easy/05-left-rotate-by-one/solution.py
def left_rotate_by_one(a):
    if not a:
        return
    first = a[0]
    for i in range(len(a) - 1):
        a[i] = a[i + 1]
    a[-1] = first


n = int(input())
a = list(map(int, input().split())) if n > 0 else []
left_rotate_by_one(a)
print(" ".join(str(x) for x in a))

File: content/learn/arrays-easy/05-left-rotate-by-one/solution.js
function leftRotateByOne(a) {
  if (a.length === 0) return;
  const first = a[0];
  for (let i = 0; i < a.length - 1; i++) {
    a[i] = a[i + 1];
  }
  a[a.length - 1] = first;
}

const n = parseInt(readline(), 10);
const a = n > 0 ? readline().split(" ").map(Number) : [];
leftRotateByOne(a);
print(a.join(" "));
